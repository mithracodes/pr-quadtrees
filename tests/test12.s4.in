144.952 -37.81 144.978 -37.79
144.953 -37.8095 144.9628 -37.7916
144.9628 -37.8095 144.97603 -37.7916
