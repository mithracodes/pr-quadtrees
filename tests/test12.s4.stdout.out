144.952 -37.81 144.978 -37.79 --> SW SW NE SW SW NW SE NW SW NW NE SE NW SE NE NE SE SE NE NE SW NW SE SE NW SW NE NW SW SE NW SW NE SE SE SW NW NE SW NE NW NE SE NW SW NW NE SE SW NW SE NW SW SW NW NE NW SE SW NW NW NE SE NW SW NW NE SW SE NW NE SE NE NW SW SE NE NW NW NE SE NE SE
144.953 -37.8095 144.9628 -37.7916 --> SW SW NE SW SW NW SE NW SW NW NE SE NW SE NE NE SE SE NE NE SW NW SE SE NW SW NE NW SW SE NW SW NE SE SE SW NW
144.9628 -37.8095 144.97603 -37.7916 --> SW NE NE SW SE SE NW NE SW NE NW NE SE NW SW NW NE SE SW NW SE NW SW SW NW NE NW SE SW NW NW NE SE NW SW NW NE SW SE NW NE SE NE NW SW SE NE NW NW NE SE NE SE
